test/rules/std_L003_L036_L039_combo_test.py

```diff
@@ -4,12 +4,14 @@
 and L039 (unnecessary white space) can result in incorrect indentation.
 """
 
+import textwrap
+
 import sqlfluff
 
 
 def test__rules__std_L003_L036_L039():
```

Comment:
One interesting thing to note here: This test started failing with this change which is to be expected. When using multiline docstrings to hold the test sql as below it is equivalent to having the SQL with a leading newline and indented e.g. 
```sql

    WITH example AS (
        SELECT
            my_id,
            other_thing,
            one_more
        FROM
            my_table
    )

    SELECT *
    FROM example
```
Agree that it's readable in the code, but to make the tests proper we can escape the first newline and use `textwrap.dedent` to remove the indents.

Reply:
I'm OK with this. As long as we've caught all the examples - is that perhaps why the DBT tests are failing? I've not looked.

Reply:
@tunetheweb It's likely why the dbt tests are failing I'm looking at the GHA logs now